powerauth-java-crypto/src/main/java/com/wultra/security/powerauth/crypto/lib/config/PowerAuthConfiguration.java

```diff
@@ -40,22 +40,37 @@ public class PowerAuthConfiguration {
     public static final int FINGERPRINT_LENGTH = 8;
 
     /**
-     * Length of signature factor in decimal formatted signature (number of decimal numbers representing signature)
+     * Length of signature factor in decimal formatted signature (number of decimal numbers representing signature, protocol v3))
      */
     public static final int SIGNATURE_DECIMAL_LENGTH = 8;
 
     /**
-     * Length of signature factor in Base64 formatted signature (number of bytes encoded to Base64)
+     * Length of signature factor in Base64 formatted signature (number of bytes encoded to Base64, protocol v3)
      */
     public static final int SIGNATURE_BINARY_LENGTH = 16;
 
     /**
-     * Number of bytes required for the signature counter data.
+     * Number of bytes required for the signature counter data (protocol v3).
      */
     public static final int SIGNATURE_COUNTER_LENGTH = 16;
 
     /**
-     * Maximum number of signature factors allowed in multi-factor signature
+     * Length of authentication code factor in Base64 formatted code (number of bytes encoded to Base64, protocol v4)
```

Comment:
Nitpick, we may use `@since`
```suggestion
     * Length of authentication code factor in Base64 formatted code (number of bytes encoded to Base64)
     *
     * @since protocol v4
```